src/cli.ts

```diff
@@ -1,5 +1,8 @@
 #!./node_modules/.bin/ts-node
 
+/* To ignore from integration and BDD testing, addressed in #354 and #368 */
```

Comment:
can somebody explain to me what is the problem with `istanbul` that we need to add `ignore file`

Reply:
I think it's because the BDD enforces code coverage. But the issue with all of these ignores, we are ignoring coverage for BDD as well as the unit tests. So perhaps this isn't the best approach.